src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -28,4 +28,14 @@
      */
     Optional<Book> findFirstByIsbn(String isbn);
 
+    /**
+     * Find a book by publisher, year and genre
+     *
+     * @param publisher book's publisher
+     * @param year      book's year
+     * @param genre     book's genre
+     * @return Book[]
+     */
+    Book[] findByPublisherAndYearAndGenre(String publisher, String year, String genre);
```

Comment:
Why are you using Book[] and User[] rather than Lists? 💭

Reply:
jajjja emmm I forgot every time